### PR TITLE
Change time calculations and time difference displays

### PR DESCRIPTION
**Pull request by TanyaPegasus**
Updated methods used to calculate time differences for /info and /whowas commands, and also changed how those difference display in embeds.

### CODE REVIEW

**Review by connorhsm — CHANGES_REQUESTED** [review]
Looks great! Much cleaner way to process the times and then display.

I've nitpicked a lot here because I know you don't have a full dev environment set up, which would usually catch a lot more of these things before a PR. Please forgive me; You have done a great job.

File: dictator/cogs/informational.py [hunk 1]
```diff
@@ -3,8 +3,9 @@
 from discord.ext import commands
 
 from db_manager import db_connection as db_conn
-import datetime
+from datetime import datetime, timezone
 import math
+import humanize
```
[the remark this answers is not in the document]
Comment (TanyaPegasus) [reply dictator/cogs/informational.py:8]: Sorry, really didn't mean to do that

File: dictator/cogs/informational.py [hunk 2]
```diff
@@ -72,14 +63,15 @@ async def info(self, interaction: discord.Interaction, user: discord.User) -> No
...
+            value=f"{math.floor(user_info[0]/60)}h {user_info[0]%60}m",
         )
         embed.add_field(name="Blocked:", value="Yes" if user_info[1] else "No")
         embed.add_field(
-            name="Joined guild:", value=member.joined_at.date() if member else "Unknown"
+            name="Joined guild:", value=f"{joined_guild}" if member else "Unknown"
         )
         embed.add_field(name="Username:", value=user_info[2])
-        embed.add_field(name="Last activity:", value=diff_formatted)
+        embed.add_field(name="Last activity:", value=f"{difference}")
```
[the remark this answers is not in the document]
Comment (TanyaPegasus) [reply dictator/cogs/informational.py:74]: We talked about that mostly in relation to the whowas command I think, but it's using the same code as this anyway. I tested a few different dates and all showed only 1 time period. I'll will test more with a greater variety of dates. I'll also look through the humanize code again. 
The logic behind what displays here is almost entirely in the humanize module. If it turns out that method won't work, we will need precisedelta, with our own logic to decide what displays when. I need to remind myself how that works, but it's likely to be quite a few more lines of code and a bit less clean. Is that preferred?

[the remark this answers is not in the document]
Comment (connorhsm) [reply dictator/cogs/informational.py:73]: Reconsidering, this is fine as is. Thanks for clarifying.

**Pull request merged**